### PR TITLE
Add markers to map

### PR DESCRIPTION
**Pull request by spioupiou**
I added the markers (and also corrected the seeds, because Kanda was showing up in Russia... I replaced with Chiyoda instead)

Screen Shot 2022-02-02 at 21 31 40

### CODE REVIEW

File: app/views/kondos/index.html.erb [hunk 1]
```diff
@@ -15,13 +15,13 @@
           path: kondo_path(kondo)
         %>
         </div>
+      <% end %>
```
Comment (spioupiou) [inline app/views/kondos/index.html.erb:18]: I put the map in the each loop!!!! Corrected now lol

**Comment by carlnoval** [comment]
I have some idea on how to resolve the conflicts but would probably be much better if you take a look @spioupiou.

File: db/seeds.rb [hunk 2]
```diff
@@ -128,12 +128,15 @@ def create_user(full_name, role)
       kondo_id: kondos[kondo_index].id,
       status: ["waiting", "confirmed", "declined", "completed"].sample,
       booked_date: Time.now + 3.days
```
[the remark this answers is not in the document]
Comment (spioupiou) [reply db/seeds.rb:130]: Damn, thanks a lot for your sharp eyes on this one! I will change and commit again

**Comment by spioupiou** [comment]
Thanks a lot for your thorough review @carlnoval! I have resolved the conflicts of both branches :)

**Comment by carlnoval** [comment]
Thank Ced, map looks goo 👍🏼 markers to, can play with locally.

**Pull request merged**